css/templates/lune/components/_sidebar.css

```diff
@@ -37,7 +45,22 @@
 .ampstart-nav-link {
   position: relative;
   display: flex;
-  justify-content: space-between;
+}
+
+/*
+ * Using margin: auto instead of justify-items: space-between, see
+ * https://stackoverflow.com/questions/36006173/exclude-element-with-fixed-positioning-from-justify-content-in-flex-layout/36102124#36102124
+ */
+.ampstart-nav-link > span {
```

Comment:
This is targeting our custom nav link configuration, where we split each letter in a separate `span` to to achieve the effect required from the design (see the `lune.navbar.nav[].link.text` field in `templates/lune/data.json`).

<img width="507" alt="screen shot 2017-07-20 at 10 11 13" src="https://user-images.githubusercontent.com/1083581/28409882-e1cf020e-6d33-11e7-80a3-877de9f3c3c0.png">

Using `.ampstart-nav-link > span` as a selector should be correct, as in our implementation we want to select the immediate `span` descendants and space them out evenly.